pool/test/filter_test.ml

```diff
@@ -5,6 +5,10 @@ let contact_email_address = "jane.doe@email.com"
 let lang = Pool_common.Language.En
 let tenant = Tenant_test.Data.full_tenant
 
+let all_experiments () =
+  Experiment.find_all Test_utils.Data.database_label () |> Lwt.map fst
```

Comment:
Use the map infix `||>` and may add a `first_experiment` function as well (change all occurrences with  `|> CCList.hd`.